Fix add-to-board for issues

### Description
It looks like sometime last week the `add-to-board` stopped working for issues. It still works for PRs.  The error it logs is:

```sh
gh: The single select option Id does not belong to the field
Error: Process completed with exit code 1.
```

[Example here](https://github.com/newrelic/node-newrelic/actions/runs/15885988304/job/44797944422).  It looks like the `gh api` call has changed. We need to triage and fix.

--- a/.github/workflows/board.yml
+++ b/.github/workflows/board.yml
@@ -24,7 +24,7 @@ on:
         type: number 
       todo_column:
         description: Name of the To-Do column in project
-        default: 'Triage Needed: Unprioritized Features'
+        default: 'Triage Needed'
         required: false
         type: string
       pr_column:

